elyra/pipeline/kfp/processor_kfp.py

```diff
@@ -532,6 +532,9 @@ def _cc_pipeline(self,
                                                                  .format(pipeline_envs['ELYRA_WRITABLE_CONTAINER_DIR'])  # noqa
                                                          })
 
+                if operation.doc:
+                    target_ops[operation.id].add_pod_annotation('elyra/user-doc', operation.doc)
```

Comment:
Could we use `elyra/node-user-doc` to make it clear that the doc string is associated with the node/pod and not the pipeline? Other annotations, such as `elyra/node-name` and `elyra/node-type` already follow that naming convention.

```suggestion
                    target_ops[operation.id].add_pod_annotation('elyra/node-user-doc', operation.doc)
```